presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java

```diff
@@ -75,6 +75,9 @@
     @Option(name = "--user", title = "user", description = "Username")
     public String user = System.getProperty("user.name");
 
+    @Option(name = "--password", title = "password", description = "Password")
```

Comment:
I think we should follow PostgreSQL and only allow the password to come from a prompt or environment variable, maybe `PRESTO_PASSWORD`